### PR TITLE
Updating readme to reflect changes in Amazon portals

### PR DESCRIPTION
**Pull request by jeroenterheerdt**
Based on my experience getting this to work, here is a stab at updating docs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,7 +47,8 @@ Note that Home Assistant includes a component (`emulated_hue`) to communicate wi
         * Select "Choose an existing role", and underneath, select `lambda_basic_execution` if it exists
         * If `lambda_basic_execution` doesn't exist, select "Create a custom role" instead, and enter `lambda_basic_execution` as the "Role Name"
     * Leave the rest of the defaults alone, and click "Next"
-    * Check "Enable event source"
+    * Using AWS CLI run the following command: `aws lambda add-permissions --function-name [name of your function, for example haaska] --statement_id 1 --action lambda:InvokeFunction --principal alexa-appkit.amazon.com --event-source-token [your skill ID]`
```
Comment (anthonylavado) [inline README.md]: In my testing of this, I found the following:
- `add-permissions` should be `add-permission`
- `--statement_id` should be `--statement-id`

**Comment by anthonylavado** [comment]
@jeroenterheerdt Nice! I made a comment regarding the `aws` line, which now that I think about it, does mean we're assuming the user has the aws cli installed and configured.

Also, I see that @mike-grant made some changes with the readme in the v3 branch, so this is perhaps best put against that?

I'm also wondering if this is perhaps best done as a Wiki entry with full, explicit steps on how to set everything up and configure it. I'm willing to put it together.

**Comment by jeroenterheerdt** [comment]
you're right - let's see what @mike-grant wants to do with this.

**Comment by mike-grant** [comment]
@anthonylavado I think that's quite a good idea, with a link from the readme to the wiki, makes things easier to deal with going forwards.

@jeroenterheerdt I'm going to accept this because it clears things up in the short term (are you able to double check the comments from @anthonylavado regarding permission/s before I accept it?), and then I'll manually merge these changes into the new v3 branch when I get 5 minutes.

File: README.md (hunk 1 above)
Comment (jeroenterheerdt) [reply README.md]: Thanks.

**Pull request merged**